examples/megacli.cpp

```diff
@@ -5983,6 +5997,22 @@ static void process_line(char* l)
 
                         return;
                     }
+                    else if (words[0] == "recentnodes")
+                    {
+                        if (words.size() == 3)
+                        {
+                            node_vector nv = client->getRecentNodes(atoi(words[2].c_str()), m_time() - 60 * 60 * atoi(words[1].c_str()), false);
+                            for (unsigned i = 0; i < nv.size(); ++i)
+                            {
+                                cout << nv[i]->displaypath() << endl;
+                            }
+                        }
+                        else
+                        {
+                            cout << "syntax error" << endl;
```

Comment:
As you can see in any other command, in case of syntax error we show the correct syntax. It's very useful.

Reply:
Yes, I agree - will PM you about this